### ISSUE TITLE
Compilation fails when contract does not contain a pragma

### ISSUE THREAD

**Issue by b-mueller** [cleanunicorn/mythos#24]
Compilation fails for contracts that don't contain "pragma solidity". For example:

```
contract Suicide {

  function kill(address payable addr) public {
    if (addr == address(0x0)) {
      selfdestruct(addr);
    }
  }
}
```

```
$ mythos analyze suicide.sol Suicide
Reading contract suicide.sol... done
Downloading Solidity version undefined
Error retrieving binary: Not Found
Compiling contract suicide.sol... failed
```

## Expected behavior

Use the latest solc if not version is explicitly specified.

**Comment by eswarasai** [comment]
@b-mueller -- This should be fixed in my PR #25 :)

**Comment by cleanunicorn** [comment]
Fixed in [0.5.0](https://github.com/cleanunicorn/mythos/releases/tag/v0.5.0)
